storage/blockchain/src/service/mod.rs

```diff
@@ -33,12 +33,12 @@
 //!
 //! ## Shutdown
 //! Upon the above handles being dropped, the corresponding thread(s) will automatically exit, i.e:
-//! - The last [`DatabaseReadHandle`] is dropped => reader thread-pool exits
-//! - The last [`DatabaseWriteHandle`] is dropped => writer thread exits
+//! - The last [`BCReadHandle`] is dropped => reader thread-pool exits
+//! - The last [`BCWriteHandle`] is dropped => writer thread exits
 //!
-//! Upon dropping the [`cuprate_database::ConcreteEnv`]:
+//! Upon dropping the [`cuprate_database::Env`]:
 //! - All un-processed database transactions are completed
-//! - All data gets flushed to disk (caused by [`Drop::drop`] impl on `ConcreteEnv`)
+//! - All data gets flushed to disk (caused by [`Drop::drop`] impl on `Env`)
```

Comment:
This doesn't make sense, the `Drop` impl is on `ConcreteEnv`.

Reply:
`ConcreteEnv` is going to be removed soon :tm:, I made this change for then. The `Drop` impl is on the environment, we document it as a guarantee that when dropped the env will be flushed to disk. 

I'll change this back for now though

Reply:
You could leave a `/// TODO: ...` after the statement.